### PR TITLE
Map http to protobuf

### PR DESCRIPTION
**Pull request by naomijub**

### CODE REVIEW

**Comment by otaviopace** [comment]
damn, she fast ⏩

File: modules/grpc-server/protos/entity.proto [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    optional int64 end_tx_id = 10;
+}
+
+message EntityHistoryResponse {
+    string content_hash = 1;
+    google.protobuf.Timestamp valid_time = 2;
+    google.protobuf.Timestamp tx_time = 3;
+    int64 tx_id = 4;
+    optional google.protobuf.Value doc = 5;
+}
```
Comment (otaviopace) [inline modules/grpc-server/protos/entity.proto:38]: No need to, but this file doesn't end with a new line like the other ones.

**Review by otaviopace — APPROVED** [review]
Well, usually on those types of PRs I tend to check field by field on the docs, but I'm not sure it's worth the effort to be honest.

First I believe you've done a good job, second we'll have to test each of the routes and requests, so I guess at that point we can double check those.

I don't know how you feel about this, but we could add a `FIXME` or a `TODO` comment in the proto files or in a `README.md` inside the folder just mentioning that we should double check the fields later.

**Comment by naomijub** [comment]
I'll add a Todo section in a README

**Pull request merged**